tests/softokn-init.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash -e
+# Copyright (C) 2022 Simo Sorce <simo@redhat.com>
+# SPDX-License-Identifier: Apache-2.0
+
+title SECTION "Setup NSS Softokn"
+
+if ! command -v certutil &> /dev/null
+then
+    echo "NSS's certutil command is required"
+    exit 0
+fi
+
+title LINE "Creating new NSS Database"
+certutil -N -d "${TOKDIR}" -f "${PINFILE}"
+
+export P11LIB="${SOFTOKNPATH%%/}/libsoftokn3${SHARED_EXT}"
+
+# NSS hardcodes the Token Label ...
+export TOKENLABEL="NSS Certificate DB"
+export TOKENLABELURI="NSS%20Certificate%20DB"
+
+# Older NSS version need a custom init arg
+#TOKENOPTIONS="pkcs11-module-init-args = configDir=${TOKDIR}"
```

Comment:
Would not it be easier to enable this line for old NSS instead of doing the skipping in the github workflow?

Reply:
There is no way to pass the argument to pkcs11-tool afaik, so it wouldn't help, I will just remove that comment.

Reply:
Indeed. My bad.